stable/airflow/templates/statefulsets-workers.yaml

```diff
@@ -111,6 +111,10 @@ spec:
             - name: dags-data
               mountPath: {{ .Values.dags.path }}
           {{- end }}
+          {{- if .Values.logsPersistence.enabled }}
```

Comment:
already in HEAD: https://github.com/helm/charts/blob/master/stable/airflow/templates/statefulsets-workers.yaml#L114